crates/noirc_evaluator/src/ssa_refactor/acir_gen/mod.rs

```diff
@@ -170,4 +176,16 @@ impl Context {
             _ => todo!(),
         }
     }
+    /// Returns an `AcirVar` that is constrained to be
```

Comment:
Incomplete comment

Reply:
Should say constrained to 'Type' -- Though currently we only constrain numeric types